packages/block/src/from-rpc.ts

```diff
@@ -61,6 +60,7 @@ function normalizeTxParams(_txParams: any) {
   // strict byte length checking
   txParams.to = txParams.to ? ethUtil.setLengthLeft(ethUtil.toBuffer(txParams.to), 20) : null
   // v as raw signature value {0,1}
-  txParams.v = txParams.v < 27 ? txParams.v + 27 : txParams.v
+  const v: number = txParams.v
+  txParams.v = v < 27 ? v + 27 : v
```

Comment:
What is happening here?

Reply:
TSLint fix. This whole method needs a refactor to remove the magic numbers and is on my to-do list.

Reply:
@holgerd77 I added a more descriptive comment to explain this magic number.

Reply:
@evertonfraga looks great, would make for a good trivia question 😄